README.md

```diff
@@ -1,77 +1,254 @@
-![](https://github.com/googleinterns/cloud-monitoring-notification-delivery-integration-sample-code/workflows/Continuous%20Integration/badge.svg)
-
 # Sample Code for the Cloud Monitoring Notification Delivery Integration Guide
```

Comment:
Thanks for doing this! Would you be able to make one more change: modify .github/workflows/continuous-integration-workflow.yml to just execute the bash scripts for the lint and pytest commands? This will make sure that the scripts don't fall out of date over time.

Here's an example: https://github.com/firebase/firebase-admin-python/blob/master/.github/workflows/ci.yml#L34

Reply:
For sure, just made the change!